galeracluster/source/documentation/galera-parameters.rst

```diff
@@ -1476,6 +1501,47 @@ This status variable is used to check which gmcast protocol version is used.
 This variable is mostly used for troubleshooting purposes and should not be implemented in a production environment.
 
 
+.. _`innodb_flush_log_at_trx_commit`:
+.. rst-class:: section-heading
+.. rubric:: ``innodb_flush_log_at_trx_commit``
+
+.. index::
+   pair: wsrep Provider Options; innodb_flush_log_at_trx_commit
+
+This variable Controls the durability/speed trade-off for commits.
+
+The possible values are:
+
+- ``0`` - Nothing is done on commit; rather the log buffer is written and flushed to the InnoDB redo log once a second. This gives better performance, but a server crash can erase the last second of transactions.
+- ``1`` - The log buffer is written to the InnoDB redo log file, and a flush to disk performed after each transaction. After a crash, committed transactions will not be lost and will be consistent with the binlog and other transactional engines. This is required for full ACID compliance.
+- ``2`` - The log buffer is written to the InnoDB redo log after each commit, but flushing takes place once a second. With this option, your system can get inconsistent and lose the last second's transactions if there is a power failure or kernel crash, but not if ``mysqld`` crashes.
+- ``3`` - Flush to disk at prepare and at commit. This option is slower and usually redundant. After a crash, committed transactions will not be lost and will be consistent with the binlog and other transactional engines. With this option, InnoDB emulates the older implementation of group commit, with 3 syncs to disk per group commit.
+

```

Comment:
Need you to make sure you're looking at 8.0/8.4 and also comparing well with mariadb (which has some changes - https://mariadb.com/kb/en/innodb-system-variables/#innodb_flush_log_at_trx_commit)

Reply:
Checked and edited.